sos/report/plugins/microshift.py

```diff
@@ -151,6 +151,8 @@ def setup(self):
         if self.path_exists('/var/lib/microshift-backups'):
             self.add_copy_spec(['/var/lib/microshift-backups/*/version',
                                 '/var/lib/microshift-backups/*.json'])
+        if self.path_exists('/var/log/kube-apiserver'):
```

Comment:
I do feel this check as a bit redundant; when this directory is not present, `add_copy_spec` will detect no file to collect either way - just bit slower than via this explicit check (but if the dir exists, we do an extra check).

Usually, we dont have these checks, while we allowed it in this plugin without an objection, as it seems.

ACK for the current version as well as for plain unconditional `add_copy_spec` one.

Reply:
Thanks for the feedback! I've removed the redundant check.